contrib/openbsm/bin/bsmconv/linau_field.c

```diff
@@ -0,0 +1,196 @@
+#include <ctype.h>
+#include <nv.h>
+#include <string.h>
+#include <stdlib.h>
+
+#include "linau_field.h"
+#include "linau_impl.h"
+#include "pjdlog.h"
+
+static size_t find_string_value_end(const char *buf, size_t start,
+    char stringtype);
+
+
+static size_t
+find_string_value_end(const char *buf, size_t start, char stringtype)
```

Comment:
It is not that simple since this function finds a `stringchar` which doesn't have `\` prepended. That's why this function can find the end of a string like `"name=\"abc def\""` and `strstr` can't.

Right?